SPECS-EXTENDED/pinfo/pinfo.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "pinfo-0.6.10.tar.bz2": "122180a0c23d11bc9eb569a4de3ff97d3052af96e32466fa62f2daf46ff61c5d"
+  "v0.6.13.tar.gz": "9dc5e848a7a86cb665a885bc5f0fdf6d09ad60e814d75e78019ae3accb42c217"
```

Comment:
Fix the source name to also indicate the package name rather than just the version string